taraxa/state/dpos/precompiled/dpos_contract.go

```diff
@@ -1016,6 +1064,14 @@ func (self *Contract) apply_genesis_entry(delegator_address *common.Address, tra
 	}
 }
 
+func (self *Contract) calculateRewardPer1Stake(rewardsPool *big.Int, stake *big.Int) *big.Int {
+	return bigutil.Div(bigutil.Mul(rewardsPool, self.cfg.MaximumStake), stake)
```

Comment:
we need assert for `MaximumStake` because 0 right now means it is disabled, or rework those functions